src/OpenTelemetry.Extensions.Docker/.publicApi/net462/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
I don't know much about PublicAPI.Shipped.txt and PublicAPI.Unshipped.txt, but it looks strange that there is
```
nullable enabled
```
here, but not in Unshipped. It feels that it should be either in both or in none.

Reply:
It is the way how VS is generating it.
It is enough to make it in one place and usually we put it to Shipped.